esp-hal/src/i2c/mod.rs

```diff
@@ -14,3 +14,10 @@ pub mod master;
 crate::unstable_module! {
     pub mod lp_i2c;
 }
+
+#[cfg(esp32s3)] // Only support ESP32-S3 for now.
+#[cfg(soc_has_rtc_i2c)]
+crate::unstable_module! {
+    /// TODO
```

Comment:
If you get to this, prefer outer comments in `rtc.rs`, don't document the module here 😅

Reply:
Whoops, forgot that TODO.

Ahhh so that's why lp_i2c was fine but rtc wasn't.